src/csv_upload/csvUpload.test.js

```diff
@@ -0,0 +1,61 @@
+import nock from 'nock'
+import {csvUpload } from './csvUpload'
+import * as funcs from './csvUpload'
+import * as api from '../exports/api'
+import {retrieve} from '../utils/books'
+import * as network from '../utils/network'
+
+describe('csvUpload', () => {
+  it('should insert to database', () =>{
+    network.insertData = jest.fn(() => {
```

Comment:
You can remove this line, since we are no longer testing the `insertData` function.